src/EA.Weee.DataAccess.Tests.Integration/ObligatedWeeeIntegrationCommon.cs

```diff
@@ -31,7 +31,8 @@ public static Return CreateReturn(Operator operatorTest, string createdById)
 
         public static Aatf CreateAatf(UKCompetentAuthority competentAuthority, Operator @operator, AatfContact contact)
         {
-            var aatf = new Aatf("aatfname", competentAuthority, "number", AatfStatus.Approved, @operator, CreateAatfAddress(null), AatfSize.Large, DateTime.Now, contact);
+            Domain.Country country = new Domain.Country(Guid.NewGuid(), "Uk");
```

Comment:
Don't do new Country - permissions on SQL server will mean this will fail in other environments - select the country as has been done in other places.

Reply:
Ok fixed where found.